### DISCUSSION TITLE
Android Enterprise deployment won't work because of package-name

### DISCUSSION THREAD

**Discussion by upits-helpdesk** [rustdesk/rustdesk#6610] [Q&A] [answered]
### Bug Description

When trying to deploy the android app of Rustdesk via Android Enterprise or specifically via Microsoft Intune which uses Android Enterprise services, we get the following error-message when trying to add the app as private app (also see screenshot):
"You need to use a different package name because com.carriez.flutter_hbb is used by a pre-installed application."

Presumably that's because the app has been in the Google playstore before.

### How to Reproduce

Add app as private app in Android Enterprise (Managed Google playstore) or something like Microsoft Intune.

### Expected Behavior

The app can be added as private app.

### Operating system(s) on local side and remote side

Android

### RustDesk Version(s) on local side and remote side

1.2.3 & 1.2.4

**Comment by upits-helpdesk** [comment]
I fail to understand why this turned into a discussion, but I would like to know if it is possible to change the packagename for the next release/nightly because we are looking into switching to Rustdesk with close to 150 Android devices so being able to deploy the app would be huge for us.

**Reply by upits-helpdesk** [reply]
@rustdesk any reply on this matter?

**Reply by RobinMJD** [reply]
Hello @rustdesk, I'm interested too by this change please.

**Comment by lecmich** [comment]
Hello @rustdesk got the same issue! Please fix this

**Comment by rustdesk** [comment, marked as answer]
https://twitter.com/rustdesk/status/1767588052325572682, custom client generator of Android (where you can modify to your own name) will come later after the Windows version.

We won't change our default package name.
